aichallenge/workspace/src/aichallenge_submit/racing_kart_description/config/vehicle_info.param.yaml

```
@@ -2,7 +2,7 @@
   ros__parameters:
     wheel_radius: 0.255
     wheel_width: 0.196
-    wheel_base: 2.14 # between front wheel center and rear wheel center
+    wheel_base: 108.7 # between front wheel center and rear wheel center
```

一部例外はありますが特に記載がない場合は単位はメートルとして処理されます。
```suggestion
    wheel_base: 1.087 # between front wheel center and rear wheel center
```

---

すみません。修正しました！